<?xml version="1.0" encoding="utf-8"?>

<launch>

<arg name="debug" default="false" />
<arg name="serial_port" default="/dev/ttyUSB0" />
<arg name="baudrate" default="2000000" />
pp
<arg name="laser_name" default="S" />
<arg name="laser_link" default="laser_link" />
<arg name="onlydistdata" default="false" />
<arg name="lensfocus" default="3.6" />

<!-- arg unless="$(arg debug)" name="launch_prefix" value="" /-->

<node name="jsk_laser_node" pkg="jsk_laser" type="jsk_laser_node" output="screen">
          <rosparam subst_value="true">
            serial_port: $(arg serial_port)
            baudrate: $(arg baudrate)
            laser_name: $(arg laser_name)
            onlydistdata: $(arg onlydistdata)
            lensfocus: $(arg lensfocus)
            laser_link: $(arg laser_link)
          </rosparam>

</node>

 <!-- node name="tf_pub_laser_link" pkg="tf" type="static_transform_publisher\
" args="0.03 -0.01 -0.10 1.5708 -0.3 -1.5708 /rhsensor /laser_tilt_link  100" / -->

<node name="laser_logger" pkg="jsk_laser" type="logger.py" output="screen">
</node>

</launch>
